permissions/views.py

```diff
@@ -0,0 +1,206 @@
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.http import JsonResponse
+from django.shortcuts import redirect, render, get_object_or_404
+from django.views.decorators.http import require_POST
+
+from yaksh.models import Course, Quiz, Lesson
+
+from .models import Permission, Role, Team
+from .utils import format_perm
+
+
+# Create your views here.
+
+
+@login_required
+def home(request):
+    # Get users
+    users = User.objects.all().exclude(username=request.user.username)
+    teams = request.user.team_members.all()
+    courses = Course.objects.filter(creator=request.user)
+
+    context = {
+        "users": users,
+        "teams": teams,
+        "courses": courses
+    }
+
+    return render(request, "home.html", context)
+
+
+@require_POST
+@login_required
+def create_team(request):
+    team_name = request.POST.get("team_name")
+    members_list = request.POST.getlist("members")
+    courses_list = request.POST.getlist("courses")
+
+    if len(team_name) > 0:
+        team = Team.objects.create(
+            name=team_name,
+            created_by=request.user
+        )
+
+        for member in members_list:
+            team.members.add(User.objects.get(username=member))
+
+        for course_id in courses_list:
+            team.courses.add(Course.objects.get(pk=course_id))
```

Comment:
No need of using for loop instead make a queryset of users and courses and add it to the team at once.